src/index.js

```diff
@@ -1,10 +1,40 @@
-import { library, dom } from '@fortawesome/fontawesome-svg-core';
-import { faArrowLeft, faRefresh } from '@fortawesome/free-solid-svg-icons';
 import './css/index.css';
-import { displayTodos } from './js/todo.js';
-
-// Using of the icon we want from fontawesome
-library.add(faRefresh, faArrowLeft);
-dom.watch();
+import {
+  displayTodos, addTodo, deleteTodo, editTodo,
+} from './modules/todo-funtionalities.js';
 
 displayTodos();
+
+const addIcon = document.querySelector('#addIcon');
+addIcon.addEventListener('click', addTodo);
+
+const form = document.querySelector('#addToDoForm');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  addTodo();
+});
+
```

Comment:
- [ ] Very good job implementing almost all the functionalities. But there is this requirement I could not find after running your project. 
- `Implement a function for editing task descriptions.`
Kindly get this functionality working as I think it is the only thing left.
![image](https://user-images.githubusercontent.com/62849872/213059500-4b529b44-4d7a-4591-b2cf-b0770783e434.png)
- As you can see from the screenshot of your project, I can create a ToDO, I can delete, but I cannot edit the description.

Reply:
Hi @t-yanick , Like the demo. If you click on the todo list text like "test 3". You will be able to edit it.  Here is the real version: https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/. And the Sneak peek: To-do list : http://savefrom.net/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DAcUd-_Yjjqg&utm_source=userjs-chrome&utm_medium=extensions&utm_campaign=link_modifier

Reply:
Hi @t-yanick ! Are you there?

Reply:
Hi @kensteph , You are right. Sorry about missing that. Kindly submit for re-review and your PR will be approved by the reviewer.

Reply:
Ok @t-yanick . No worries! Thank you